environments/icds-cas/public.yml

```diff
@@ -28,6 +28,8 @@ swift_url: https://mowcdmum.ipstorage.tatacommunications.com/auth/v1.0/
 nofile_limit: 65536
 
 redis_appendonly: 'yes'
+redis_maxmemory: 6gb
```

Comment:
are there any catches in ansible that make sure this is < 1/2 the system's memory?

Reply:
I searched our code as well as the redis galaxy module and I don't see anything of the sort; it's only mentioned a couple times in code, and only to interpolate it in a template.